configure

```diff
@@ -3486,6 +3512,30 @@ if test -n "$_host"; then
 			_seq_midi=no
 			_timidity=no
 			;;
+		kos32)
+		        SDL_CFLAGS="-I${KOS32_SDK_DIR}/sources/SDL-1.2.2_newlib/include"
+			SDL_LIBS="-lSDLn -lsound"
+			_pkgconfig=/bin/false
+			_pkg_config=no
+
+			if test "$_freetype2" != no; then
+			    FREETYPE2_LIBS="-lfreetype.dll"
+			    FREETYPE2_STATIC_LIBS="-lfreetype.dll"
+			    # neither pkg-config nor freetype-config work, so we setup freetype manually
+			    _freetypepath="${KOS32_SDK_DIR}/sources/freetype/"
+			    FREETYPE2_CFLAGS="-I${KOS32_SDK_DIR}/sources/freetype/include"
+			    _freetype_found="true"
+			fi
+			PNG_LIBS="-lpng16.dll -lz.dll"
+			PNG_CFLAGS="-I${KOS32_SDK_DIR}/sources/libpng"
+			_sdl=yes
```

Comment:
What about actually checking these for presence and bailing out?

Reply:
It already happens just further in the file